students/Jaidjen/lesson04/Trigrams.py

```diff
@@ -0,0 +1,60 @@
+s= 'I wish I may I wish I might'
+words = 'I wish I may I wish I might'.split()
+
+def build_trigrams(words):
+
+    trigrams = dict()
+    trigrams['I wish'] = ['I']
+    trigrams['wish I'] = ['may']
+    trigrams['I may'] = ['I']
+    trigrams['may I'] = ['wish']
+    trigrams['I wish'].append('I')
+    trigrams['wish I'].append('might')
+
+    print("This is a trigrams")
+    return trigrams
```

Comment:
you're not using `words` in here - if this function was implemented to create trigrams from words list then you could have easily updated to feed in data from a a file.